include/openthread/dataset.h

```diff
@@ -152,10 +152,10 @@ enum
 };
 
 /**
- * This type represents Channel Mask Page 0.
+ * This type represents Channel Mask Page.
  *
  */
-typedef uint32_t otChannelMaskPage0;
+typedef uint32_t otChannelMaskPage;
```

Comment:
Is the intention to encode two channel pages (2.4 GHz and sub-GHz) in this single mask? If so, I'd rename to `otChannelMask`. Otherwise, we should figure out how to encode multiple channel page masks in this structure.

Reply:
Yes, I want to encode two channel page masks in one single mask to save memory. I will update the name.